health/notifications/hangouts/README.md

```diff
@@ -0,0 +1,33 @@
+# Hangouts Chat
+
+This is what you will get:
+![Netdata on Hangouts](https://imgur.com/a/FVlZ3F0)
+You need:
```

Comment:
Put an empty line above this.

Reply:
And maybe clarify that you need to set these up in Google Hangouts, not Netdata? Someone might get confused. One possibility:

```suggestion
To receive notifications in Google Hangouts, you need the following in your Hangouts setup:
```